README.md

```diff
@@ -54,7 +54,9 @@ The history command gives a historical view of a given secret. This view is usef
 $ chamber exec <service...> -- <your executable>
 ```
 
-The purpose of this command is to be used inside your service's docker container. Exec sets up environment variables populated with all the latest versions of secrets for the given service and environment.
+Exec executes the given command in an environment where all secrets from the specified services are set.  Secret keys are converted to upper case (for example a secret with key `secret_key` will become `SECRET_KEY`).
```

Comment:
How about `exec populates the environment with the secrets from the specified services and executes the given command. Secret keys....`?